event-protocol/schemas/defs.json

```diff
@@ -38,6 +38,49 @@
         "type": "string"
       },
       "location": { "$ref": "defs.json#/location" }
-    }
+    },
+    "required": [
+      "name",
+      "location"
+    ],
+    "additionalProperties": false
+  },
+  "ast-location": {
+    "type": "object",
+    "properties": {
+      "line": {
+        "type": "integer",
+        "minimum": 1
+      },
+      "column": {
+        "type": "integer",
+        "minimum": 1
+      }
+    },
+    "required": [
+      "line",
+      "column"
+    ],
+    "additionalProperties": false
+  },
+  "ast-tag": {
```

Comment:
I don't understand why we need this extra def. The only difference from the other `tag` def I can see is the extra `type` attribute which is always set to `Tag`. Can't we just get rid of that in the source data and then we can use one tag structure for pickles and ast elements?